DOCS/man/input.rst

```diff
@@ -925,9 +925,9 @@ Remember to quote string arguments in input.conf (see `Flat command syntax`_).
     only be set on the key specified by ``name``.
 
 ``keybind <name> <command>``
```

Comment:
change it also here

Reply:
Wouldn't it make sense to do it in this commit? I don't want to be this pedantic, so it is fine, just going through commits it wasn't visible.